bigflow/__init__.py

```diff
@@ -16,20 +17,21 @@
     'JobContext',
     'Definition',
     'Config',
-    'resources'
 ]
 
-try:
-    from . import bigquery
-    __all__.append('bigquery')
-except ImportError:
-    pass
-
-try:
-    from . import monitoring
-    __all__.append('monitoring')
-except ImportError:
-    pass
+
+def __getattr__(name):
+    if name in {
+        'bigquery',
+        'monitoring',
+        'resources',
+    }:
+        importlib.import_module(f"bigflow.{name}")
+        msg = f"Don't use `bigflow.{name}` directly, add explicit import `import bigflow.{name}` instead"
```

Comment:
I don't get it. Can you give an example of a wrong import, in that message?

Reply:
I've changed message.
Also added small workaround to check way of importing. Now it should not print warning on statements like `from bigflow import dataproc`.

Reply:
ok 👍